src/symbols/Line.tsx

```diff
@@ -38,6 +38,11 @@ export interface LineProps {
    */
   dashed?: boolean;
 
+  /**
+   * The size of the dash.
+   */
+  dashSize?: number;
```

Comment:
i actually wonder if the dash size should be an array .. like how svgs handle dashed stroke lines:
https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Attribute/stroke-dasharray

not sure we need to go crazy with the option, but maybe just `[number, number]` to represent the dash size and the gap?

thoughts?